polybar-scripts/inbox-reddit/inbox-reddit.sh

```diff
@@ -1,10 +1,31 @@
 #!/bin/sh
+#
+# This script retrieves unread messages from your Reddit account using a JSON
+# link.
+#
+# To use this script, make sure you have the following dependencies installed:
```

Comment:
The dependencies are already part of the `inbox-reddit/README.md`.
We shouldn't repeat us. `font-awesome` is no dependency.

Help texts should be in the README.md not in the script.